samples/app-dev/Connectors/OnPremise/TestViaEndpoint/readme.md

```diff
@@ -0,0 +1,147 @@
+
+./tibagent configure agent -p <pass_differnt_port> <agent_name2> 
+
+./tibagent configure connect -a <access_key> -s <access_secret> <agent_name2>
+
+./tibagent start agent --spec container_port:onpremise_host:onpremise_port <agent_name2>
+
+To know more about Hybrid agents. Please refer [here](https://integration.cloud.tibco.com/docs/#tci/using/hybrid-agent/agent-command-reference.html?TocPath=Using%2520TIBCO%2520Cloud%25E2%2584%25A2%2520Integration%257CUsing%2520the%2520TIBCO%2520Cloud%25E2%2584%25A2%2520Integration%2520-%2520Hybrid%2520Agent%257C_____5)
+
+## Import the sample
+
+1. Download the sample json file i.e., PostgreSQL-Onprem-Flogotester.json.
+
+2. Create a new empty app
+
+![Create an app](../../../import-screenshots/sqlserver_screenshot/1.png)
+
+3. On the app details page, select import app option.
+
+![Select import](../../../import-screenshots/sqlserver_screenshot/2.png)
+
+4. Now click on ‘browse to upload’ button and select the app.json from your machine that you want to import.
+
+![Import your sample](../../../import-screenshots/Onpremise_Postgresql/3.png)
+
+5. Click on Upload Button. The Import app dialog displays shows what Triggers is using, Flow name, Connection and some generic errors and warnings as well as any specific errors or warnings pertaining to the app you are importing.
+
+![The Import app dialog](../../../import-screenshots/Onpremise_Postgresql/4.jpg)
+
+![The Import app dialog](../../../import-screenshots/Onpremise_Postgresql/5.jpg)
+
+6. After importing app is done, in connection tab make sure to re-enter the password and click on connect button to establish the connection.
+
+## Understanding the configuration
+
+### The Connection
+
+When you import the app you need to re-enter the password and establish the connection.
+
+![The connection](../../../import-screenshots/Onpremise_Postgresql/6.png)
+
+![The connection](../../../import-screenshots/Onpremise_Postgresql/7.png)
+
+![The connection](../../../import-screenshots/Onpremise_Postgresql/8.png)
+
+In the connection, note that,
+1. Host - In this field we give private ip of the on-premise system on which database is hosted.
+2. On-premise "True" means that it establishes connection to on-premise database.
+
+### The Flow
+
+If you go inside the app, you can see in flow we have 4 activities (Query,Insert, Update and Delete) that perform some operations.
+Also in flow we have Log Message and Return activity for getting the output.
+
+![Sample Response](../../../import-screenshots/Onpremise_Postgresql/9.jpg)
+
+### Run the application
+
+In order to test the on-premise DBConnector app via endpoint we need to attach access key in app and push the app.
+In app , click on Environment controls -> click on Hybrid connectivity -> select access key from drop down -> click on Push updates.
+ 
+
+![sample Response](../../../import-screenshots/Onpremise_Postgresql/15.jpg)
+
+![Sample Response](../../../import-screenshots/Onpremise_Postgresql/16.jpg)
+
+
```

Comment:
Please add the section to run the on premise flogo tester as well. 
1. How to enable the flogotester from UI/API
2. run the flogotester for the on premise DB app for one of the flow.

Reply:
For On-premise flogo tester i have added separate sample.
Please check flogo tester sample under /OnPremise/TestInFlowTester